src/types/SIOP.types.ts

```diff
@@ -127,6 +127,13 @@ export interface DiscoveryMetadataOpts {
   subjectTypesSupported?: SubjectType[] | SubjectType;
   idTokenSigningAlgValuesSupported?: KeyAlgo[] | KeyAlgo;
   requestObjectSigningAlgValuesSupported?: SigningAlgo[] | SigningAlgo;
+  dids_supported: boolean;
```

Comment:
Camelcase in the opts, snake in the payload

Reply:
Fixed!